src/components/mgt-people/mgt-people.ts

```diff
@@ -8,7 +8,9 @@
 import * as MicrosoftGraph from '@microsoft/microsoft-graph-types';
 import { customElement, html, property, TemplateResult } from 'lit-element';
 import { repeat } from 'lit-html/directives/repeat';
+import { BetaGraph } from '../../BetaGraph';
```

Comment:
Can we avoid importing BetaGraph in people and person by importing it in graph.presence.ts instead?

Reply:
I think it's awkward at either location... I don't really have a preference. This would be a good @shweaver-MSFT question.